activerecord/lib/active_record/attribute_methods/serialization.rb

```diff
@@ -11,6 +11,15 @@ module ClassMethods
         # serialized object must be of that class on assignment and retrieval.
         # Otherwise <tt>SerializationTypeMismatch</tt> will be raised.
         #
+        # Keep in mind that database adapters handle certain serialization tasks
+        # for you. For instance: +json+ and +sonb+ types in PostgreSQL will be
```

Comment:
I fixed the typo in the merge commit.

On Fri, May 29, 2015, 8:23 AM Kasper Timm Hansen <notifications@github.com>
wrote:

> In activerecord/lib/active_record/attribute_methods/serialization.rb
> <https://github.com/rails/rails/pull/20364#discussion_r31328436>:
>
> > @@ -11,6 +11,15 @@ module ClassMethods
> >          # serialized object must be of that class on assignment and retrieval.
> >          # Otherwise <tt>SerializationTypeMismatch</tt> will be raised.
> >          #
> > +        # Keep in mind that database adapters handle certain serialization tasks
> > +        # for you. For instance: +json+ and +sonb+ types in PostgreSQL will be
>
> Typo in +jsonb+
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/20364/files#r31328436>.
>